app/views/finders/_facet_tags.html.erb

```diff
@@ -11,9 +11,9 @@
       <div class="facet-tags__group">
         <% applied_filter.each do |filter| %>
           <div class="facet-tags__wrapper">
-            <p class="facet-tags__preposition"><%= filter[:preposition] %></p>
-            <div class="facet-tag">
-              <p class="facet-tag__text"><%= filter[:text] %></p>
+            <span class="facet-tags__preposition"><%= filter[:preposition] %></span>
+            <span class="facet-tag">
+              <span class="facet-tag__text"><%= filter[:text] %></span>
```

Comment:
This has brought together the first half of the sentence but it still breaks up at the "and" where a new div is introduced:
 About -subject x-
 and -subject y-

Reply:
I decided to do it this way because there's no chance the 'remove' button could be confused with the wrong bit of text. I think it also makes it more readable. Each option is clearly separated by a new line and we don't end up with a very long block of text where line wrapping could lead to confusion.

Reply:
I'd suggest taking it back to Anika in this case, my understanding is that it should be grouped as a whole sentence in the same way for assistive tech that it is visually but I understand your point too.

Reply:
Anika seems happy with this approach. Are you okay to approve?

Reply:
Sure thing.